### PR TITLE
migrate to live-1

### PR DESCRIPTION
**Pull request by asmega**

### CODE REVIEW

File: app/services/adapters/amazon_ses_adapter.rb [hunk 1]
```diff
@@ -19,7 +19,7 @@ def self.send_mail( opts = {} )
     private
 
     def self.client
-      Aws::SES::Client.new
+      Aws::SES::Client.new(region: 'eu-west-1')
```
Comment (solidgoldpig) [inline app/services/adapters/amazon_ses_adapter.rb]: Ouch.

Is it worth adding a comment explaining why this is still pointing at Ireland?

File: deploy/fb-submitter-chart/templates/network_policy.yaml [hunk 2]
```diff
@@ -54,3 +54,20 @@ spec:
     ports:
     - protocol: TCP
       port: 3000
+---
+apiVersion: networking.k8s.io/v1
```
Comment (solidgoldpig) [inline deploy/fb-submitter-chart/templates/network_policy.yaml]: What does this network policy do? Is this punching a hole out for the west-1 email? Or something else? I don't understand what the cidr is doing :(

File: app/services/adapters/amazon_ses_adapter.rb (hunk 1 above)
Comment (asmega) [reply app/services/adapters/amazon_ses_adapter.rb]: done

File: deploy/fb-submitter-chart/templates/network_policy.yaml (hunk 2 above)
Comment (asmega) [reply deploy/fb-submitter-chart/templates/network_policy.yaml]: i just retested and seems to work without it, so i've removed it. I can't quite remember the reason why I added but it did at one stage not work without it

**Pull request merged**